lib/gris/caching.rb

```diff
@@ -0,0 +1,21 @@
+module Gris
+  module Caching
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def cache_key_for(id)
```

Comment:
It could be private, but it's public for now because there is one test of it. Happy to make it private.